lib/daru/view/adapters/googlecharts/display.rb

```diff
@@ -163,6 +167,65 @@ def to_js_spreadsheet(data, element_id=SecureRandom.uuid)
       js << "\n</script>"
       js
     end
+
+    private
+
+    def apply_formatters
+      return unless user_options && user_options[:formatters]
+      @formatters = []
+      user_options[:formatters].each_value do |formatter|
+        frmttr = case formatter[:type].to_s.capitalize
+                 when 'Color'
+                   apply_color_formatter(formatter)
+                 when 'Pattern'
+                   apply_pattern_formatter(formatter)
+                 else
+                   apply_remaining_formatter(formatter)
+                 end
+        @formatters << frmttr
+      end
+    end
+
+    def apply_color_formatter(formatter)
```

Comment:
Please write one example, so that others can understand what are the options/parameter is required for this specific formatter and why  there is separate method for this formatter. 

Write down docs above all the methods.

Reply:
Done.